Code/Framework/AzCore/AzCore/Component/EntitySerializer.cpp

```diff
@@ -206,7 +206,9 @@ namespace AZ
         {
             if (component)
             {
-                componentMapOut.emplace(AZStd::string::format("Component_[%llu]", component->GetId()), component);
+                AZStd::string componentAlias = component->GetSerializedIdentifier();
+                AZ_Assert(!componentAlias.empty(), "Component alias is empty.");
```

Comment:
It'd be good to have a fallback solution of ("Component_[%llu]")in case the alias is empty so that serialization doesn't stop/break.

Reply:
OK, I'll add a fallback.